Userland/Services/WebDriver/Client.cpp

```diff
@@ -593,6 +594,19 @@ ErrorOr<JsonValue, HttpError> Client::handle_get_element_attribute(Vector<String
     return make_json_value(result);
 }
 
+// 12.4.3 Get Element Property, https://w3c.github.io/webdriver/#dfn-get-element-property
+// GET 	/session/{session id}/element/{element id}/property/{name}
+ErrorOr<JsonValue, HttpError> Client::handle_get_element_property(Vector<StringView> const& parameters, JsonValue const& payload)
+{
+    dbgln_if(WEBDRIVER_DEBUG, "Handling GET /session/<session_id>/element/<element_id>/property/<name>");
+    auto* session = TRY(find_session_with_id(parameters[0]));
+
+    // NOTE: Spec steps handled in Session::get_element_property().
+    auto result = TRY(session->get_element_property(payload, parameters[1], parameters[2]));
+
+    return make_json_value(result);
```

Comment:
Please use the new style without that comment :)

```suggestion
    dbgln_if(WEBDRIVER_DEBUG, "Handling GET /session/<session_id>/element/<element_id>/property/<name>");
    auto* session = TRY(find_session_with_id(parameters[0]));
    auto result = TRY(session->get_element_property(payload, parameters[1], parameters[2]));
    return make_json_value(result);
```